### DISCUSSION TITLE
Possibility to fill the user email in the username field

### DISCUSSION THREAD

**Discussion by smartyash88** [jamf/Setup-Manager#54] [Q&A]
Is it possible to have the user input their email address and have that email looked up using cloud identity provider and have all the mapped fields filled in User & Location tab?

**Comment by kevinandrews** [comment]
Even if could pull the user name without the domain and confirmation, I'd find that really helpful.

**Comment by scriptingosx** [comment]
Accessing this data is not possible without using the Jamf Pro API.

We have intentionally avoided using the Jamf Pro API because you would have to provide API credentials in the Setup Manager and those would be readable by every other user and process on the system.
